### ISSUE TITLE
placeholder label option should not be recommended as label

### ISSUE THREAD

**Issue by stevefaulkner** [w3c/html#736]
from the spec:

> If a select element has a required attribute specified, does not have a multiple attribute specified, and has a display size of 1; and if the value of the first option element in the select element’s list of options (if any) is the empty string, and that option element’s parent node is the select element (and not an optgroup element), then that option is the select element’s placeholder label option.

>If a select element has a required attribute specified, does not have a multiple attribute specified, and has a display size of 1, then the select element must have a placeholder label option.
http://w3c.github.io/html/sec-forms.html#ref-for-placeholder-label-option-1

>select element’s placeholder label option. A placeholder label option does not represent an actual option, but instead represents a label for the select control.
http://w3c.github.io/html/sec-forms.html#ref-for-placeholder-label-option-3

> If a select element contains a placeholder label option, the user agent is expected to render that option in a manner that conveys that it is a label, rather than a valid option of the control. This can include preventing the placeholder label option from being explicitly selected by the user. When the placeholder label option’s selectedness is true, the control is expected to be displayed in a fashion that indicates that no valid option is currently selected.
http://w3c.github.io/html/rendering.html#ref-for-placeholder-label-option-5

It does not appear to be implemented as described in the spec and as it suffers from the same problem as the placeholder attribute (text is only visible until a selection has been made or data entered).

Also note that it is not included in the select elements accessible name calculation although maybe it should, as fallback.

**Comment by spanchang** [comment]
I agree. It answers the question I raised initially: 
https://lists.w3.org/Archives/Public/w3c-wai-gl/2016OctDec/0559.html
Thanks,
Sailesh

**Comment by dylanb** [comment]
Is `label` a defined term in the HTML spec in the same way as it is in WCAG 2?

The reason I ask is because `label` under WCAG 2 refers to visible labels whereas `accessible name` refers to the text available to AT.

If the HTML spec has the same definition, then the text as it is is ok because the placeholder label option is simply serving as a visible label. It will still be necessary to either associate that label with the select or use some other mechanism to generate an accessible name.

Also, the text is always visible to the user without changing their input selection. You can press SPACE to display the full list of options and scroll around. So in practice, it behaves differently than the placeholder attribute.

**Comment by LJWatson** [comment]
Ping @stevefaulkner?

**Comment by chaals** [comment]
That text is no longer in the spec, and what is left matches implementation in Firefox / Blink (Edge differs, in that it will not allow submission of an empty value where the others will. @travisleithead ?)

Closing this issue, if necessary will follow up the Edge divergence.

(The text could use an edit - it is pretty hard to understand, but I'll bounce that to another day).
